site/content/docs/user/quick-start.md

```diff
@@ -330,36 +330,8 @@ Feature gates are a set of key=value pairs that describe alpha or experimental f
 {{< codeFromInline lang="yaml" >}}
 kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
-# patch the generated kubeadm config with a featuregate
-kubeadmConfigPatches:
-- |
-  kind: ClusterConfiguration
-  metadata:
-    name: config
-  apiServer:
-    extraArgs:
-      "feature-gates": "FeatureGateName=true"
-  scheduler:
-    extraArgs:
-      "feature-gates": "FeatureGateName=true"
-  controllerManager:
-    extraArgs:
-      "feature-gates": "FeatureGateName=true"
-- |
-  kind: InitConfiguration
-  metadata:
-    name: config
-  nodeRegistration:
-    kubeletExtraArgs:
-      "feature-gates": "FeatureGateName=true"
-- |
-  kind: KubeletConfiguration
-  featureGates:
-    FeatureGateName: true
-- |
-  kind: KubeProxyConfiguration
-  featureGates:
-    FeatureGateName: true
+featureGates:
```

Comment:
this is the config used in the GA jobs
```yaml
# config for 1 control plane node and 2 workers (necessary for conformance)
kind: Cluster
apiVersion: kind.x-k8s.io/v1alpha4
networking:
  ipFamily: ipv4
nodes:
- role: control-plane
- role: worker
- role: worker
featureGates: {"AllAlpha":false,"AllBeta":false,"RotateKubeletClientCertificate":true}
```

I have doubts now, better ask @BenTheElder directly :sweat_smile:

Reply:
@aojea as far as I see, that's the same data structure expressed in two different ways. (just key: value)

Reply:
that's my understanding, but I prefer to defer when I'm not sure 😉

Reply:
this is just yaml differences.